src/v2/guide/syntax.md

```diff
@@ -58,7 +58,7 @@ El contenido del `span` se reemplazará con el valor de la propiedad `rawHtml`,
 
 <p class="tip">La ejecución dinámica de HTML arbitrario en su sitio web puede ser muy peligrosa porque puede conducir fácilmente a [vulnerabilidades de XSS](https://en.wikipedia.org/wiki/Cross-site_scripting). Utilice solo la interpolación HTML en contenido de confianza y **nunca** en contenido proporcionado por usuario.</p>
 
-### Attributos
+### Atributos
 
 Los bigotes no se pueden utilizar dentro de los atributos HTML. En su lugar, use una [directiva v-bind](../api/#v-bind):

```

Comment:
Hi @marcorichetta, let me check with the team about the translation for mustaches. Maybe we can change it within your changes. @Tintef Do we have a better translation than `bigotes`?

Reply:
@marcorichetta if you have a suggestion feel free to post it here

Reply:
I really don't like using `bigotes` to be honest. Let me search for some better examples

Reply:
@sxcamacho I think we should leave mustaches, and type in italic as we do with other english words. If necessary, we could add a clarification saying that "mustaches" are `corchetes dobles`.

Here's an example:
```
Los _mustaches_ (doble corchetes) no se pueden utilizar dentro de los atributos HTML.
```

Just like the guys from the [Portuguese translation](https://br.vuejs.org/v2/guide/syntax.html#Texto) did.

What do you think?

Reply:
Hi! 
The original suggestion I made is about the replacement of `postfijo` with `sufijo`.

The `bigotes` translation it's kind of funny :laughing: and I'm fine with that, but if you're looking for a better translation, why not `llaves`?

Reply:
Yes, it's funny but it isn't a common phrase in Spanish. In this kind of case, we're adding the original word in English and the translation or the idea. I agree with you, `llaves` is correct too. In that case, we could translate this phrase as @Tintef says but using `llaves` or `llaves doble`. 

```suggestion
Los _mustaches_ (llaves doble) no se pueden utilizar dentro de los atributos HTML. En su lugar, use una [directiva v-bind](../api/#v-bind):
```

Reply:
Sorry, I replied without @Tintef's answer being load.
I agree and it's also better for consistency between translations.